db/migrate/20240815004215_create_event.rb

```diff
@@ -0,0 +1,14 @@
+class CreateEvent < ActiveRecord::Migration[7.1]
+  def change
+    create_enum :type, ['public', 'private', 'friends only']
+    create_table :events do |t|
+      t.string :name, default: '', null: false
+      t.string :description
+      t.enum :event_type, enum_type: 'type', null: false
+      t.datetime :open_time
+      t.datetime :close_time
```

Comment:
You need this to reference a location. Where is the event going to be? 
![image](https://github.com/user-attachments/assets/0a1e30ff-7041-4ebe-b5e1-aa9ca08283b8)